test/e2e/util.go

```diff
@@ -284,15 +285,15 @@ func waitResourcesDeleted(t *testing.T, f *framework.Framework, c *spec.Cluster)
 }
 
 func waitBackupDeleted(f *framework.Framework, c *spec.Cluster) error {
-	err := retryutil.Retry(5*time.Second, 5, func() (done bool, err error) {
-		rl, err := f.KubeClient.Extensions().ReplicaSets(f.Namespace).List(k8sutil.ClusterListOpt(c.Metadata.Name))
-		if err != nil {
-			return false, err
-		}
-		if len(rl.Items) > 0 {
+	err := retryutil.Retry(5*time.Second, 5, func() (bool, error) {
+		_, err := f.KubeClient.AppsV1beta1().Deployments(f.Namespace).Get(k8sutil.BackupSidecarName(c.Metadata.Name), metav1.GetOptions{})
+		if err == nil {
 			return false, nil
 		}
-		return true, nil
+		if apierrors.IsNotFound(err) {
+			return true, nil
+		}
+		return false, err
 	})
 	if err != nil {
 		return fmt.Errorf("failed to wait backup RS deleted: %v", err)
```

Comment:
Change `failed to wait backup RS deleted` to `failed to wait backup Deployment deleted`